ui-modules/home/app/views/about/about.template.html

```diff
@@ -93,4 +91,7 @@ <h4 class="list-group-item-heading">{{feature.name}}</h4>
             </ul>
         </div>
     </div>
+
+    <h2>Logbook</h2>
+    <br-logbook></br-logbook>
```

Comment:
Open to suggestions on how to hide/display the logbook if needed, e.g. return back the button, accordion, etc.